tests/containers/nodejs/Dockerfile

```
@@ -1,5 +1,7 @@
 ARG NODE_IMAGE_TAG
-FROM node:${NODE_IMAGE_TAG}
+ARG NODE_IMAGE
```

```suggestion
ARG NODE_IMAGE=node
```
then we don't need to add `NODE_IMAGE` everywhere :)